index.html

```diff
@@ -1151,169 +1367,182 @@ <h2>Methods</h2>
-                    <a>[[\QuicTransportStreams]]</a> internal slot.</li>
-                  </ol>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <pre class="idl">
+        [ Exposed=Window ]
+        interface RTCQuicBidirectionalStream {
+            readonly attribute unsigned long long streamId;
+            readonly attribute RTCQuicTransport transport;
+        };
+        RTCQuicBidirectionalStream includes RTCQuicWritableStream;
+        RTCQuicBidirectionalStream includes RTCQuicReadableStream;
+        </pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCQuicBidirectionalStream" data-dfn-for="RTCQuicBidirectionalStream" class=
+          "attributes">
+            <dt><dfn><code>streamId</code></dfn> of type <span class=
+            "idlAttrType"><a>unsigned long long</a></span>, readonly</dt>
+            <dd>
+              <p>The readonly attribute referring to the ID of the
+              <code><a>RTCQuicBidirectionalStream</a></code> object.</p>
+            </dd>
+            <dt><dfn><code>transport</code></dfn> of type <span class=
+            "idlAttrType"><a>RTCQuicTransport</a></span>, readonly</dt>
+            <dd>
+              <p>The readonly attribute referring to the related <code><a>RTCQuicTransport</a></code> object.</p>
+            </dd>
+          </dl>
+       </section>
+      </div>
+    </section>
+    <section id="rtcquicsendstream-interface-definition*">
+      <h3>Interface <dfn>RTCQuicSendStream</dfn></h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+        interface RTCQuicSendStream {
+            readonly attribute unsigned long long streamId;
+            readonly attribute RTCQuicTransport transport;
+        };
+        RTCQuicSendStream includes RTCQuicWritableStream;
+        </pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCQuicSendStream" data-dfn-for="RTCQuicSendStream" class=
+          "attributes">
+            <dt><dfn><code>streamId</code></dfn> of type <span class=
+            "idlAttrType"><a>unsigned long long</a></span>, readonly</dt>
+            <dd>
+              <p>The readonly attribute referring to the ID of the
+              <code><a>RTCQuicSendStream</a></code> object.</p>
+            </dd>
+            <dt><dfn><code>transport</code></dfn> of type <span class=
+            "idlAttrType"><a>RTCQuicTransport</a></span>, readonly</dt>
+            <dd>
+              <p>The readonly attribute referring to the related <code><a>RTCQuicTransport</a></code> object.</p>
+            </dd>
+          </dl>
+       </section>
+      </div>
+    </section>
+    <section id="rtcquicreceivestream-interface-definition*">
+      <h3>Interface <dfn>RTCQuicReceiveStream</dfn></h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+        interface RTCQuicReceiveStream {
+            readonly attribute unsigned long long streamId;

```

Comment:
Now that we're dealing with 4 types of streams (remotely created bidi, locally created bidi, locally created uni, remotely created uni), does this stream ID include the two bits that encode that enum on the wire in QUIC, or does the streamId not?  I could see arguments for either way.

Reply:
I think it would be nice to include those two bits. It's gives the developer less of an abstraction on top of the QUIC library, and also gives information on what type of stream it is (if they care). I don't think the extra information harms a web developer much (unless they assume streams will be ordered 1,2,3. etc.). What are the counter arguments?

Reply:
Well... I sat down and thought of some arguments but they all seemed dumb because I realized the JS can just do "stream.streamId >> 2" if they care. 

So, let's just use the whole stream ID.